src/NewTwitchApi/Auth/AuthGuzzleClient.php

```diff
@@ -8,8 +8,12 @@
 
 class AuthGuzzleClient extends Client
 {
-    private const BASE_URI = 'https://id.twitch.tv/oauth2/';
+    const BASE_URI = 'https://id.twitch.tv/oauth2/';
 
+    /**
+     * AuthGuzzleClient constructor.
+     * @param array $config
```

Comment:
We don't need `@param` annotations for method arguments. These are already defined in the method arguments themselves with type hints.

Reply:
agreed, i'll remove them